Taglierino.hs

```diff
@@ -571,6 +603,7 @@ data Program =
   Program { pEvents        :: S.Set Term
           , pAllowed       :: S.Set Term
           , pPublic        :: S.Set Term
+          , pStore         :: M.Map String (S.Set Term) 
```

Comment:
If the store is part of the agent declaration, maybe we could create a new `AgentDef` data type for holding the agent's code _and_ its table.

Reply:
Yes, this would be better. Maybe I could it in the future.